### PR TITLE
tf_upgrade_v2 for network/classification/resnet.py

### PR DESCRIPTION
**Pull request by ytfksw**
## What this patch does to fix the issue.

I converted only the blueoil/network/classification/resnet.py with the command line tool tf_upgrade_v2 to run on tensorflow2 because there are a lot of targets in the network folder.

Here's what I did:

install newest tenosrflow (tensorflow-gpu==1.5.2)
Run the upgrade script tf_upgrade_v2
Check the upgrade report for warnings and errors
Run test (make test)

## Link to any relevant issues or pull requests.
* #479 
* https://www.tensorflow.org/guide/upgrade

### CODE REVIEW

**Comment by ytfksw** [comment]
This is the whole convert log:
```
TensorFlow 2.0 Upgrade Script
-----------------------------
Converted 1 files
Detected 0 issues that require attention
--------------------------------------------------------------------------------
================================================================================
Detailed log follows:

================================================================================
--------------------------------------------------------------------------------
Processing file 'networks/classification/resnet.py'
 outputting to 'networks/classification/resnet.py'
--------------------------------------------------------------------------------

92:25: INFO: Added keywords to args of function 'tf.nn.avg_pool'
92:25: INFO: Renamed keyword argument for tf.nn.avg_pool from value to input
92:25: INFO: Renamed 'tf.nn.avg_pool' to 'tf.nn.avg_pool2d'
98:25: INFO: Added keywords to args of function 'tf.pad'
173:21: INFO: Renamed 'tf.Print' to 'tf.compat.v1.Print'
173:39: INFO: Added keywords to args of function 'tf.shape'
173:57: INFO: Added keywords to args of function 'tf.argmax'
174:22: INFO: Renamed 'tf.Print' to 'tf.compat.v1.Print'
174:41: INFO: Added keywords to args of function 'tf.shape'
174:60: INFO: Added keywords to args of function 'tf.argmax'
176:25: INFO: Added keywords to args of function 'tf.reduce_sum'
181:29: INFO: Added keywords to args of function 'tf.reduce_mean'
--------------------------------------------------------------------------------
```

**Comment by ytfksw** [comment]
tf_upgrade_v2 sometimes changed argment to keyword argument.
I deleted the keyword argument and checking it one by one (https://github.com/blue-oil/blueoil/pull/1069/commits/f172aa58b15123d143b0a91189244d0f9d73a5ec), because the arguments become meaninglessly long .

Because API has changed by changing from tf1 to tf2.
It seems to have been converted to a keyword argument to avoid misunderstanding.

avg_pool
tf1:https://www.tensorflow.org/versions/r1.15/api_docs/python/tf/nn/avg_pool
```
tf.nn.avg_pool(
    value,
    ksize,
    strides,
    padding,
    data_format='NHWC',
    name=None,
    input=None
)
```

tf2:https://www.tensorflow.org/api_docs/python/tf/nn/avg_pool2d
```
tf.nn.avg_pool2d(
    input, ksize, strides, padding, data_format='NHWC', name=None
)
```


tf.pad
tf1:https://www.tensorflow.org/versions/r1.15/api_docs/python/tf/pad
```
tf.pad(
    tensor,
    paddings,
    mode='CONSTANT',
    name=None,
    constant_values=0
)
```

tf2:https://www.tensorflow.org/api_docs/python/tf/pad
```
tf.shape(
    tensor, paddings, mode='CONSTANT', constant_values=0, name=None
)
```

tf.shape
tf1:https://www.tensorflow.org/versions/r1.15/api_docs/python/tf/shape
```
tf.shape(
    input,
    name=None,
    out_type=tf.dtypes.int32
)
```

tf2:https://www.tensorflow.org/api_docs/python/tf/shape
```
tf.shape(
    input, out_type=tf.dtypes.int32, name=None
)
```

tf.argmax

tf1:https://www.tensorflow.org/versions/r1.15/api_docs/python/tf/math/argmax
```
tf.math.argmax(
    input,
    axis=None,
    name=None,
    dimension=None,
    output_type=tf.dtypes.int64
)
```

tf2:https://www.tensorflow.org/api_docs/python/tf/math/argmax
```
tf.math.argmax(
    input, axis=None, output_type=tf.dtypes.int64, name=None
)
```


tf.reduce_sum
tf1:https://www.tensorflow.org/versions/r1.15/api_docs/python/tf/math/reduce_sum
```
tf.math.reduce_sum(
    input_tensor,
    axis=None,
    keepdims=None,
    name=None,
    reduction_indices=None,
    keep_dims=None
)
```
tf2:https://www.tensorflow.org/api_docs/python/tf/math/reduce_sum
```
tf.math.reduce_sum(
    input_tensor, axis=None, keepdims=False, name=None
)
```

tf.reduce_mean
tf1:https://www.tensorflow.org/versions/r1.15/api_docs/python/tf/math/reduce_mean
```
tf.math.reduce_mean(
    input_tensor,
    axis=None,
    keepdims=None,
    name=None,
    reduction_indices=None,
    keep_dims=None
)
```

tf2:https://www.tensorflow.org/api_docs/python/tf/math/reduce_mean
```
tf.math.reduce_mean(
    input_tensor, axis=None, keepdims=False, name=None
)
```

File: blueoil/networks/classification/resnet.py [hunk 1]
```diff
@@ -170,8 +170,14 @@ def loss(self, softmax, labels):
...
-            labels = tf.Print(labels, [tf.shape(labels), tf.argmax(labels, 1)], message="labels:", summarize=200)
-            softmax = tf.Print(softmax, [tf.shape(softmax), tf.argmax(softmax, 1)], message="softmax:", summarize=200)
+            labels = tf.compat.v1.Print(labels,
+                                        [tf.shape(labels), tf.argmax(labels, axis=1)],
+                                        message="labels:",
+                                        summarize=200)
+            softmax = tf.compat.v1.Print(softmax,
+                                         [tf.shape(softmax), tf.argmax(softmax, axis=1)],
+                                         message="softmax:",
+                                         summarize=200)
```
Comment (tfujiwar) [inline blueoil/networks/classification/resnet.py:180]: What about this? It's more consistent with the other function calls in this file.
```py
labels = tf.compat.v1.Print(
    labels,
    [tf.shape(labels), tf.argmax(labels, axis=1)],
    message="labels:",
    summarize=200
)
```

Comment (ytfksw) [reply blueoil/networks/classification/resnet.py]: Thank you.  OK. I fixed it for consistency!

**Pull request merged**